packages/theme/src/theme/_dropdowns.scss

```diff
@@ -16,10 +16,10 @@ $dropdown-caret-up-axis: -45deg !default;
 			margin: 0 0 $dropdown-caret-size $dropdown-caret-size;
 			height: $dropdown-caret-size;
 			width: $dropdown-caret-size;
-			border-width: 1px !important;
-			border-style: solid;
+			border: 1px solid;
 			border-bottom-color: transparent;
 			border-left-color: transparent;
+			box-shadow: 1px -1px 0px;
```

Comment:
remove 0px by 0

Reply:
no surge demo ?

Reply:
nope, issue with travis. Waiting for them to fix, for now we removed the demos in #1548

Reply:
This PR revert the deactivation, it will me merged when travis is green again : #1549